modules/sdk-coin-atom/test/unit/utils.ts

```diff
@@ -0,0 +1,43 @@
+import should from 'should';
+import { address, blockHash, txIds } from '../resources/atom';
+import utils from '../../src/lib/utils';
+
+describe('utils', () => {
+  it('should validate addresses correctly', () => {
+    should.equal(utils.isValidAddress(address.address1), true);
+    should.equal(utils.isValidAddress(address.address2), true);
+    should.equal(utils.isValidAddress(address.address3), false);
+    should.equal(utils.isValidAddress(address.address4), true);
+    should.equal(utils.isValidAddress('dfjk35y'), false);
+    should.equal(utils.isValidAddress(undefined as unknown as string), false);
+    should.equal(utils.isValidAddress(''), false);
+  });
+
+  it('should validate block hash correctly', () => {
+    should.equal(utils.isValidBlockId(blockHash.hash1), true);
+    should.equal(utils.isValidBlockId(blockHash.hash2), true);
+    // param is coming as undefined so it was causing an issue
+    should.equal(utils.isValidBlockId(undefined as unknown as string), false);
+    should.equal(utils.isValidBlockId(''), false);
```

Comment:
Seems to be covered in the `should validate invalid block hash correctly` block below